feat: added support for proxy agent in https call (#30)
lpinca/shopify-token: README.md

  token that respects the user's permission level when making API requests
  (called online access). This is strongly recommended for embedded apps.
  Defaults to offline access mode.

> - `agent` - Optional - An http/https agent which will be passed on to https
>   request made for obtaining auth token. This is useful when trying to obtain
>   token from a server which has constraints on internet access.

Reviewer: ```suggestion
- `agent` - Optional - An HTTPS agent which will be passed to the HTTPS
  request made for obtaining the auth token. This is useful when trying to
  obtain a token from a server that has restrictions on internet access.
```

Author: Updated the README.md
